package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "test:unit": "NODE_ENV=test mocha --timeout 999999 --colors --exit tests/unit/*",
     "test:integration": "NODE_ENV=test mocha --timeout 999999 --colors --exit tests/integration/*",
+    "test:withTag": "NODE_ENV=test mocha --timeout 999999 --colors --recursive --exit --grep newTest tests/*",
```

Comment:
Que penses-tu de l'appeler `test:tag` et que le # que l'on mette ce soit #tag ?

Reply:
je suis d'accord :)

Reply:
Le problème c'est qu'il ne fait pas la différence entre `#tag` et `un test avec le mot tag dedans`, donc cette commande exécute le test `should return a 400 if no tagging` par exemple 😄 

Le # c'est une bonne pratique pour trouver le tag rapidement quand tu veux le supprimer mais il n'est pas indispensable

Reply:
Ca ne fonctionne pas si tu mets le # dans le grep ?

Reply:
Mais avec @tag ça a fonctionné 😎

Reply:
En soit j'ai pris l'idée de `@` sur l'article qui parlait de cette bonne pratique : il donnait en exemple @ et # comme façons d'identifier les tags que l'on ajoute.
Donc on peut prendre celui que vous préférez !
👍 - @
👌 - #